common/artifacts/000_wc_artifacts_rings_generic.txt

```diff
@@ -0,0 +1,390 @@
+### Ring ###
+
+# Legendary
+
+ring_5_diplomacy = {
+	quality = 5
+	flags = { ring }
+
+	active = { }
+
+	general_opinion = 1
+	vassal_opinion = 1
+	same_religion_opinion = 2
+	church_opinion = 5
+	sex_appeal_opinion = 16
+
+	monthly_character_prestige = 1
+
+	picture = "GFX_inv_misc_6oring_bluelv4"
+
+	slot = ring
+}
+
+ring_5_martial = {
+	quality = 5
+	flags = { ring }
+
+	active = { }
+
+	combat_rating = 25
+
+	monthly_character_prestige = 1
+
+	levy_reinforce_rate = 0.15
+	land_morale = 0.15
+	land_organisation = 0.15
+
+	picture = "GFX_inv_misc_6oring_redlv4"
+
+	slot = ring
+}
+
+ring_5_stewardship = {
+	quality = 5
+	flags = { ring }
+
+	active = { }
+
+	build_cost_city_modifier = -0.25
+	build_time_castle_modifier = -0.25
+	build_cost_temple_modifier = -0.25
+	build_time_temple_modifier = -0.25
+
+	monthly_character_prestige = 1
+
+	picture = "GFX_inv_misc_6oring_greenlv4"
+
+	slot = ring
+}
+
+ring_5_intrigue = {
+	quality = 5
+	flags = { ring }
+	
+	active = { }
+
+	assassinate_chance_modifier = 0.2
+	arrest_chance_modifier = 0.4
+	plot_power_modifier = 0.05
+	murder_plot_power_modifier = 0.1
+	plot_discovery_chance = 0.05
+
+	monthly_character_prestige = 1
+
+	picture = "GFX_inv_misc_6oring_purplelv4"
+
+	slot = ring
+}
+
+ring_5_learning = {
+	quality = 5
+	flags = { ring }
+
+	active = { }
+
+	fertility = 0.4
+	health = 0.2
+
+	monthly_character_prestige = 1
+	monthly_character_piety = 0.25
+
+	picture = "GFX_inv_misc_6oring_silverlv4"
+
+	slot = ring
+}
+
+# Epic
+
+ring_4_diplomacy = {
+	quality = 4
+	flags = { ring }
+	
+	active = { }
+
+	general_opinion = 1
+	vassal_opinion = 1
+	same_religion_opinion = 2
+	church_opinion = 4
+	sex_appeal_opinion = 13
+
+	monthly_character_prestige = 0.8
+
+	picture = "GFX_inv_misc_6oring_bluelv3"
+
+	slot = ring
+}
+
+ring_4_martial = {
+	quality = 4
+	flags = { ring }
+	
+	active = { }
+
+	combat_rating = 20
+
+	monthly_character_prestige = 0.8
+
+	levy_reinforce_rate = 0.12
+	land_morale = 0.12
+	land_organisation = 0.12
+
+	picture = "GFX_inv_misc_6oring_redlv3"
+
+	slot = ring
+}
+
+ring_4_stewardship = {
+	quality = 4
+	flags = { ring }
+	
+	active = { }
+
+	build_cost_city_modifier = -0.2
+	build_time_castle_modifier = -0.2
+	build_cost_temple_modifier = -0.2
+	build_time_temple_modifier = -0.2
+
+	monthly_character_prestige = 0.8
+
+	picture = "GFX_inv_misc_6oring_greenlv3"
+
+	slot = ring
+}
+
+ring_4_intrigue = {
+	quality = 4
+	flags = { ring }
+
+	active = { }
+
+	assassinate_chance_modifier = 0.16
+	arrest_chance_modifier = 0.32
+	plot_power_modifier = 0.04
+	murder_plot_power_modifier = 0.08
+	plot_discovery_chance = 0.04
+
+	monthly_character_prestige = 0.8
+
+	picture = "GFX_inv_misc_6oring_purplelv3"
+
+	slot = ring
+}
+
+ring_4_learning = {
+	quality = 4
+	flags = { ring }
+
+	active = { }
+
+	fertility = 0.32
+	health = 0.16
+
+	monthly_character_prestige = 0.8
+	monthly_character_piety = 0.2
```

Comment:
Same as for the T5-amulet, too much prestige+piety, otherwise perfectly balanced.